tsconfig.json

```diff
@@ -45,6 +45,7 @@
     },
     "pretty": true,
     "resolveJsonModule": true,
+    "skipLibCheck": true,
```

Comment:
Are there any TS issues introduced with this update?

Reply:
yeah I saw an error where [path-scurry](https://www.npmjs.com/package/path-scurry) doesn't play nicely with the node typings:

```
node_modules/path-scurry/dist/commonjs/index.d.ts:117:31 - error TS2720: Class 'PathBase' incorrectly implements class 'Dirent'. Did you mean to extend 'Dirent' and inherit its members as a subclass?
  Property 'parentPath' is missing in type 'PathBase' but required in type 'Dirent'.

117 export declare abstract class PathBase implements Dirent {
                                  ~~~~~~~~

  node_modules/@types/node/fs.d.ts:247:9
    247         parentPath: string;
                ~~~~~~~~~~
    'parentPath' is declared here.


Found 1 error in node_modules/path-scurry/dist/commonjs/index.d.ts:117
```

Reply:
I kind of like having `skipLibCheck` on because it does skip a bunch of type-checking and makes builds faster

Reply:
just tested and I think it makes calling `tsc` around 3 seconds faster to have the option on

Reply:
Makes sense to me 👍